tests/Feature/WPTitleWPHeadTest.php

```diff
@@ -95,17 +101,27 @@ function _assert_all_meta( $description ) {
 <meta name='description' content='{$description}' /><!-- WP SEO -->
 <meta name='demo arbitrary title' content='demo arbitrary content' /><!-- WP SEO -->
 EOF;
-		$this->assertSame( strip_ws( $expected ), strip_ws( get_echo( array( WP_SEO(), 'wp_head' ) ) ) );
+
+		ob_start();
+		WP_SEO()->wp_head();
+		$html = ob_get_clean();
```

Comment:
[Mantle has `Utils::get_echo()`](https://github.com/alleyinteractive/mantle-framework/blob/353a7584bebe8413ddd8e883aef05a1a8d6bbcb8/src/mantle/testing/class-utils.php#L69)

Reply:
Aha.
I copied this pattern from the tests Greg already migrated,so we might need to revisit those too.

Reply:
attn @mogmarsh

Reply:
Updated!